src/pages/index.tsx

```diff
@@ -40,12 +62,16 @@ export default function Home() {
                 <p className='text-xl font-semibold'>日付を入力</p>
               </div>
               <p className='ml-4 text-sm'>※候補日時を選択</p>
-              <div>ここにカレンダー</div>
+              <Calender
+                onChange={(selectedDate: number[]) => {
+                  setValue('dates', selectedDate)
+                }}
+              />
             </div>
           </div>
         </div>
-        <Button>調整ちゃんを入力</Button>
-      </div>
+        <Button type='submit'>調整ちゃんを入力</Button>
```

Comment:
せっかくバリデーションを作ったので

useFormの宣言部分を
```
  const {
    register,
    formState: { errors, isValid, isSubmitting },
    handleSubmit,
    setValue,
    watch,
  } = useForm<Schedule>({
    mode: 'onSubmit',
  })
```

として

```suggestion
        <Button
          type='submit'
          disabled={!isValid || !calenderValid}
          loading={isSubmitting}
        >
          調整ちゃんを入力
        </Button>```

としたら、validがかかってる時はボタンが押せなくなって、送信中の時はローディングが入ります
ローディングはonSubmitを非同期処理にしなくちゃいけないので

```
  const router = useRouter()
  const onSubmit = async (data: Schedule) => {
    // 送信処理
    console.log(data)
    await router.push('/schedule') // 遷移先のURL
  }
```
とすればちょっとだけローディング入ります
Toastとかも出せるといいね！

Reply:
[419f942](https://github.com/NUTFes/chosei-chan/pull/46/commits/419f942fda6c683507e3a39cbecebf58d9d2539d)
toast通知はページ遷移すると消えてしまう問題を修正しなければいけない。
けど、時間かかりそうだったので一旦未実装で他のタスクします